src/components/public/PlayBar/styles.ts

```diff
@@ -74,4 +73,58 @@ width:200px;
 display:flex;
 justify-content:space-between;
 align-items:center;
+`
+
+export const PlayBarContainer = styled.div`
+width:580px;
+height:20px;
+display:flex;
+justify-content:space-between;
+align-items:center;
+margin-top:15px;
+`
+
+export const Time = styled.span`
+color:#AFAFAF;
+font-size:13px;
+`
+
+export const PlayBar = styled.div`
+width:480px;
+height:100%;
+display:flex;
+align-items:center;
+cursor: pointer;
+position:relative;
+:hover .ControlCircle{

```

Comment:
.ControlCircle -> ${ControlCircle} 
and controlCircle should be on top of PlayBar